app/models/nivs_report.rb

```diff
@@ -0,0 +1,36 @@
+require "csv"
+
+class NivsReport
+  EXPECTED_HEADERS = %w[
```

Comment:
Why not simply `HEADERS`? `EXPECTED_HEADERS` makes it feel like we are in a test.